patches/build_gn.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/third_party/widevine/cdm/BUILD.gn b/third_party/widevine/cdm/BUILD.gn
+index e5fdb29e858f..01b6973cb494 100644
+--- a/third_party/widevine/cdm/BUILD.gn
++++ b/third_party/widevine/cdm/BUILD.gn
+@@ -2,6 +2,7 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//chromiumcontent/config.gni")
+ import("//build/config/chrome_build.gni")
+ import("//build/config/features.gni")
+ import("//chrome/version.gni")  # TODO layering violation
+@@ -93,7 +94,7 @@ if (widevine_cdm_binary_files != []) {
+       "//build/config/sanitizers:deps",
+     ]
+ 
+-    if (is_mac) {
++    if (is_mac && !is_electron_build) {
+       ldflags = [
+         # Not to strip important symbols by -Wl,-dead_strip.
+         "-Wl,-exported_symbol,_PPP_GetInterface",
```

Comment:
for mac you will need `-Wl,-install_name,@loader_path/libwidevinecdm.dylib`
gyp handled it correctly, but there is a comment in the gn file `#TODO(jrummell) Mac: 'DYLIB_INSTALL_NAME_BASE': '@loader_path',`